account-kit/react/src/components/auth/card/loading/otp.tsx

```diff
@@ -1,4 +1,4 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useEffect' is defined but never used.
<!-- __reviewdog__:ChAxYzQ3YjhkMDNjYjA2MjBjEgZyZGpzb24= -->